modules/catch2/metadata.json

```diff
@@ -11,6 +11,7 @@
         "github:catchorg/Catch2"
     ],
     "versions": [
```

Comment:
Please keep this sorted.

Reply:
Thank you for the quick review.
I guess you are referring to an ascending order.
Changed it accordingly